setup.py

```diff
@@ -22,13 +13,15 @@
 
 setup(
     name="django-bootstrap4",
-    version=VERSION,
-    url="https://github.com/zostera/django-bootstrap4",
-    author="Dylan Verheul",
-    author_email="dylan@dyve.net",
     description="""Bootstrap support for Django projects""",
     long_description=readme + "\n\n" + history,
-    license="BSD",
+    long_description_content_type="text/x-rst",
+    author="Dylan Verheul",
+    author_email="dylan@dyve.net",
+    url="https://github.com/zostera/django-bootstrap4",
+    license="BSD-3-Clause",
+    use_scm_version=True,
```

Comment:
Interesting, from a maintainers perspective. But also if I'm not mistaken: version is not explicitly available anymore in the package source...

Reply:
Yes, is that a problem? Version is a tag.

Reply:
I think not, since everything is installed using pip so `pip freeze` will tell the version installed.